### PR TITLE
Add client admin details to property view

### PR DESCRIPTION
**Pull request by MBenedicte**
## Description ##

Display the client admin details in the property details
## Type of change ##
Please select the relevant option
- [x] Bug fix(a non-breaking change which fixes an issue)
- [ ] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of a feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [ ] Integration test
- [x] unit test

## Checklist: ##
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

## How can this PR be manually tested: ##

After cloning the branch, view one property

## Related Pivotal Tracker stories ##

168453220

### CODE REVIEW

**Review by Jaman-dedy — APPROVED** [review]
Good stuff @MBenedicte

File: property/renderers.py [hunk 1]
```diff
@@ -32,7 +32,9 @@ def single_property_format(self, data):
                 'client_name': client.client_name,
                 'phone': client.phone,
                 'email': client.email,
-                'address': client.address
+                'address': client.address,
+                'admin_email': client.client_admin.email,
+                'admin_id': client.client_admin.id
```
Comment (caleb-42) [inline property/renderers.py:37]: I can see you are getting the client admin's id so we can use it to check against the currently logged in user's id and then decide whether or not to show the edit or delete buttons for a specific property....... nice work @MBenedicte

**Review by patrickf949 — APPROVED** [review]
Nice

**Review by caleb-42 — APPROVED** [review]

**Review by Niyitangasam — APPROVED** [review]
LGTM

**Review by Victoradukwu — APPROVED** [review]
Works as expected.

**Pull request merged**